store/mongo/datastore_mongo_test.go

```diff
@@ -14,185 +14,190 @@
-					Name:     "acmeHub",
-				},
-			},
-			Error: context.Canceled,
-		},
-	}
-	for i := range testCases {
-		tc := testCases[i]
-		t.Run(tc.Name, func(t *testing.T) {
-			db.Wipe()
-			mgo := db.Client()
-			ds := NewDataStoreWithClient(mgo)
-			err := ds.SetSettings(tc.CTX, tc.Settings)
-			if tc.Error != nil {
-				if assert.Error(t, err) {
-					assert.Regexp(t, tc.Error.Error(), err.Error())
-				}
-			} else {
-				assert.NoError(t, err)
-				idty := identity.FromContext(tc.CTX)
-				var tenantID string
-				if idty != nil {
-					tenantID = idty.Tenant
-				}
-				fltr := bson.D{{
-					Key: "tenant_id", Value: tenantID,
-				}}
-
-				var doc bson.Raw
-				err := mgo.Database(DbName).
-					Collection(CollNameSettings).
-					FindOne(tc.CTX, fltr).
-					Decode(&doc)
-				if !assert.NoError(t, err) {
-					t.FailNow()
-				}
-
-				field := doc.Lookup(KeyTenantID)
-				actualTID, ok := field.StringValueOK()
-				assert.True(t, ok, "bson document does not contain tenant_id field")
-				assert.Equal(t, tenantID, actualTID)
-
-				var settings model.Settings
-				bson.Unmarshal(doc, &settings)
-				assert.Equal(t, tc.Settings, settings)
-			}
-		})
-	}
-}
-
-func TestGetSettings(t *testing.T) {
-	const tenantID = "123456789012345678901234"
-	testCases := []struct {
-		Name string
-
-		CTX context.Context
-
-		Settings model.Settings
-		Error    error
-	}{
-		{
-			Name: "ok got settings",
-			CTX: identity.WithContext(context.Background(), &identity.Identity{
-				Tenant: tenantID,
-			}),
-			Settings: model.Settings{
-				ConnectionString: &model.ConnectionString{
-					HostName: "localhost",
-					Key:      []byte("password123"),
-					Name:     "acmeHub",
-				},
-			},
-		},
-		{
-			Name: "no settings for tenant",
-			CTX: identity.WithContext(context.Background(), &identity.Identity{
-				Tenant: "111111111111111111111111",
-			}),
-		},
-		{
-			Name: "error, context deadline exceeded",
-			CTX: func() context.Context {
-				ctx, cancel := context.WithCancel(context.Background())
-				cancel()
-				return ctx
-			}(),
-			Error: context.Canceled,
-		},
-	}
-	for i := range testCases {
-		tc := testCases[i]
-		t.Run(tc.Name, func(t *testing.T) {
-			db.Wipe()
-			client := db.Client()
-			collAlerts := client.
-				Database(DbName).
-				Collection(CollNameSettings)
-
-			ctx := identity.WithContext(context.Background(), &identity.Identity{
-				Tenant: tenantID,
-			})
-
-			_, err := collAlerts.InsertMany(ctx, []interface{}{
-				mstore.WithTenantID(ctx, tc.Settings),
-			})
-			assert.NoError(t, err)
-
-			db := NewDataStoreWithClient(client)
-			settings, err := db.GetSettings(tc.CTX)
-			if tc.Error != nil {
-				if assert.Error(t, err) {
-					assert.Regexp(t,
-						tc.Error.Error(),
-						err.Error(),
-						"error did not match expected expression",
-					)
-				}
-			} else {
-				assert.NoError(t, err)
-				assert.Equal(t, tc.Settings, settings)
-			}
-		})
-	}
-}
+// func TestCreateIntegration(t *testing.T) {
```

Comment:
Why are these tests commented? Will you update them with a follow-up PR?

Reply:
ohh, I missed those. I'll fix them in this PR if it's OK?